back/db/migrate/20211806161357_drop_page_links.rb

```diff
@@ -0,0 +1,9 @@
+class DropPageLinks < ActiveRecord::Migration[6.1]
```

Comment:
It has the same number `20211806161357` as `AddDeletedAtToTenants`, so it won't be run. I'm wondering how it could happen. Why didn't you use `rails g migration ...` to generate this file?

It seems we use increment for this number. But I cannot understand why?

Reply:
It's safe to use it now. Even if a new number is less than the previous one, the migration will be run.

Reply:
I think it's because a version number got wrong at some point, wrote a Slack post here: https://citizenlabco.slack.com/archives/C016C2EHURY/p1638894402014500